config/crd/bases/consul.hashicorp.com_proxydefaults.yaml

```diff
@@ -43,7 +43,8 @@ spec:
           properties:
             config:
               description: Config is an arbitrary map of configuration values used by Connect proxies. Any values that your proxy allows can be configured globally here. Supports JSON config values. See https://www.consul.io/docs/connect/proxies/envoy#configuration-formatting
-              type: Any
+              format: byte
```

Comment:
The new version of the controller-tools changes this. `Any` was a bug actually: https://github.com/kubernetes-sigs/controller-tools/pull/505. It was never supported. Our CRD in helm use `object`. I've modified the hack script to swap this when copying over to `consul-helm`.